includes/Integrations/Core_Themes_Support.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Class Core_Themes_Support
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Integrations;
+
+use Google\Web_Stories\Customizer;
+
+/**
+ * Class Core_Themes_Support.
+ */
+class Core_Themes_Support {
+
+	/**
+	 * Default array of core themes to add support to.
+	 *
+	 * @var array
+	 */
+	public static $supported_themes = [
+		'twentytwentyone',
+		'twentytwenty',
+		'twentynineteen',
+		'twentyseventeen',
+		'twentysixteen',
+		'twentyfifteen',
+		'twentyfourteen',
+		'twentythirteen',
+		'twentytwelve',
+		'twentyeleven',
+		'twentyten',
+		'classic',
+		'default',
+	];
+
+	/**
+	 * Adds theme support for Web Stories.
+	 *
+	 * This will enable add_theme_support with predefined
+	 * options supported themes.
+	 *
+	 * @since 1.3.0
+	 *
+	 * @return void
+	 */
+	public function extend_theme_support() {
+		add_theme_support( 'web-stories' );
+	}
```

Comment:
>  "This will enable add_theme_support with predefined options supported themes."

Should we actually have some individual predefined options for each theme? Every theme might need different sensible default values.

Thoughts?

Reply:
We are adding support for only `Circles` view by default and it has very limited options.
I don't think we'll need more specific defaults for these core themes then what are defined already. 😅 

Unless we find a use case to have different size of circles on some themes. This could be one example.